egs/lre07/v1/run.sh

```diff
@@ -14,31 +14,32 @@ mfccdir=`pwd`/mfcc
 vaddir=`pwd`/mfcc
 languages=local/general_lr_closed_set_langs.txt
 
+data_root=/export/corpora/LDC
 # Training data sources
-local/make_sre_2008_train.pl /export/corpora5/LDC/LDC2011S05 data
-local/make_callfriend.pl /export/corpora/LDC/LDC96S60 vietnamese data
-local/make_callfriend.pl /export/corpora/LDC/LDC96S59 tamil data
-local/make_callfriend.pl /export/corpora/LDC/LDC96S53 japanese data
-local/make_callfriend.pl /export/corpora/LDC/LDC96S52 hindi data
-local/make_callfriend.pl /export/corpora/LDC/LDC96S51 german data
-local/make_callfriend.pl /export/corpora/LDC/LDC96S50 farsi data
-local/make_callfriend.pl /export/corpora5/LDC/LDC96S48 french data
-local/make_callfriend.pl /export/corpora5/LDC/LDC96S49 arabic.standard data
-local/make_callfriend.pl /export/corpora5/LDC/LDC96S54 korean data
-local/make_callfriend.pl /export/corpora5/LDC/LDC96S55 chinese.mandarin.mainland data
-local/make_callfriend.pl /export/corpora5/LDC/LDC96S56 chinese.mandarin.taiwan data
-local/make_callfriend.pl /export/corpora5/LDC/LDC96S57 spanish.caribbean data
-local/make_callfriend.pl /export/corpora5/LDC/LDC96S58 spanish.noncaribbean data
+local/make_sre_2008_train.pl $data_root/LDC2011S05 data
+local/make_callfriend.pl $data_root/LDC96S60 vietnamese data
+local/make_callfriend.pl $data_root/LDC96S59 tamil data
+local/make_callfriend.pl $data_root/LDC96S53 japanese data
+local/make_callfriend.pl $data_root/LDC96S52 hindi data
+local/make_callfriend.pl $data_root/LDC96S51 german data
+local/make_callfriend.pl $data_root/LDC96S50 farsi data
+local/make_callfriend.pl $data_root/LDC96S48 french data
+local/make_callfriend.pl $data_root/LDC96S49 arabic.standard data
+local/make_callfriend.pl $data_root/LDC96S54 korean data
+local/make_callfriend.pl $data_root/LDC96S55 chinese.mandarin.mainland data
+local/make_callfriend.pl $data_root/LDC96S56 chinese.mandarin.taiwan data
+local/make_callfriend.pl $data_root/LDC96S57 spanish.caribbean data
+local/make_callfriend.pl $data_root/LDC96S58 spanish.noncaribbean data
 local/make_lre96.pl /export/corpora/NIST/lid96e1 data
```

Comment:
It turns out that preparing LRE 1996 is redundant, since it is included with LDC2006S31 which is prepared next. Can you remove this line? The results will _not_ be affected since the data/train_unsplit is the same either way.